umpost/um2netcdf.py

```diff
@@ -736,6 +733,34 @@ def fix_level_coord(cube, z_rho, z_theta, tol=1e-6):
                 c_sigma.var_name = 'sigma_theta'
 
 
+MAX_NP_INT32 = np.iinfo(np.int32).max
+MIN_NP_INT32 = np.iinfo(np.int32).min
+
+
+def convert_32_bit(cube):
+    """
+    Convert 64 bit int/float data to 32 bit (in place).
+
+    Parameters
+    ----------
+    cube : iris.cube object to modify.
```

Comment:
```suggestion
    cube : iris.cube object to modify.

Warns
-----
    TBD : If the input cube has values beyond 32-bit limits, causing an overflow.
```

See comment below on the warning to replace TBD.

Reply:
Good idea, it's fixed in 9bd6f270.

Reply:
This is fixed too.